drivers/sensor/st/lsm6dso/lsm6dso_trigger.c

```diff
@@ -48,6 +48,193 @@ static int lsm6dso_enable_t_int(const struct device *dev, int enable)
 }
 #endif
 
+#if defined(CONFIG_LSM6DSO_TILT)
+
+static int lsm6dso_enable_tilt_int(const struct device *dev, int enable)
+{
+	const struct lsm6dso_config *cfg = dev->config;
+	stmdev_ctx_t *ctx = (stmdev_ctx_t *)&cfg->ctx;
+	int ret = 0;
+	lsm6dso_emb_sens_t sens;
+
+	sens.tilt = PROPERTY_ENABLE;
```

Comment:
should this be as below? this way you save power when turning off interrupt by disabling more than just the routing?

```suggestion
	sens.tilt = enable;
```